meta-lmp-bsp/recipes-bsp/u-boot/u-boot-fio-mfgtool/imx8mmevk/lmp.cfg

```diff
@@ -1,39 +1,62 @@
+CONFIG_SPL_DM=y
 CONFIG_SPL_FIT=y
 CONFIG_SPL_FIT_SIGNATURE=y
 CONFIG_SPL_LOAD_FIT=y
+# CONFIG_SPL_FIT_IMAGE_TINY is not set
+CONFIG_SPL_OF_CONTROL=y
+CONFIG_SPL_CRYPTO_SUPPORT=y
+CONFIG_SPL_HASH_SUPPORT=y
+CONFIG_SPL_USB_HOST_SUPPORT=y
+CONFIG_SPL_USB_GADGET=y
+CONFIG_SPL_USB_SDP_SUPPORT=y
+CONFIG_RSA=y
+CONFIG_HASH=y
+CONFIG_SYS_MMCSD_RAW_MODE_U_BOOT_SECTOR=0x300
 # CONFIG_SPL_LEGACY_IMAGE_SUPPORT is not set
 # CONFIG_SPL_EFI_PARTITION is not set
+# CONFIG_SPL_DM_USB is not set
 # CONFIG_SPL_PARTITION_UUIDS is not set
 # CONFIG_SPL_DOS_PARTITION is not set
 # CONFIG_CMD_DEKBLOB is not set
 CONFIG_OF_LIBFDT_OVERLAY=y
 CONFIG_FIT=y
 CONFIG_FIT_SIGNATURE=y
 CONFIG_FIT_SIGNATURE_STRICT=y
+CONFIG_SPL_FIT_SIGNATURE=y
+CONFIG_SPL_FIT_SIGNATURE_STRICT=y
 CONFIG_FIT_VERBOSE=y
 # CONFIG_LEGACY_IMAGE_FORMAT is not set
+CONFIG_IMX_HAB=y
+# CONFIG_SPL_IMX_HAB is not set
 CONFIG_PARTITION_UUIDS=y
+CONFIG_TEE=y
+CONFIG_OPTEE=y
 CONFIG_FIOVB=y
 CONFIG_CMD_FIOVB=y
 CONFIG_CMD_XIMG=y
 CONFIG_SUPPORT_EMMC_RPMB=y
-CONFIG_SUPPORT_EMMC_BOOT=y
 CONFIG_USB=y
 CONFIG_DM_USB=y
 CONFIG_USB_FUNCTION_FASTBOOT=y
+CONFIG_USB_GADGET_MASS_STORAGE=y
 CONFIG_USB_GADGET_MANUFACTURER="FSL"
 CONFIG_USB_GADGET_VENDOR_NUM=0x0525
 CONFIG_USB_GADGET_PRODUCT_NUM=0xa4a5
 CONFIG_FASTBOOT_FLASH=y
 CONFIG_FASTBOOT_UUU_SUPPORT=y
 CONFIG_FASTBOOT_BUF_ADDR=0x42800000
 CONFIG_FASTBOOT_BUF_SIZE=0x40000000
+CONFIG_FSL_FASTBOOT_BOOTLOADER2=y
+CONFIG_FSL_FASTBOOT_BOOTLOADER2_OFFSET=0x300
```

Comment:
Quick question, how can we using mfgtools write two sets of bootloader images into boot0 (for regular/recovery paths) with this approach? If the offset is provided here, and it can not be adjustable in runtime

Reply:
I think we'll need to tackle flashing of backup images in a separate PR.  For now this is needed to flash the original u-boot.itb binary.